meshtastic/node_runtime/settings_runtime.py

```
@@ -0,0 +1,608 @@
+            self._node._raise_interface_error(
+                "startOTA only possible on local node"
+            )  # noqa: SLF001
+
+        # COMPAT_STABLE_SHIM: support legacy keyword aliases used by older callers:
+        # `ota_mode` -> `mode`, and `ota_hash`/`hash` -> `ota_file_hash`.
+        legacy_hash = extra_kwargs.pop("hash", None)
+        if extra_kwargs:
+            unexpected = ", ".join(sorted(extra_kwargs))
+            raise TypeError(
+                f"startOTA() got unexpected keyword argument(s): {unexpected}"
+            )
```

Ruff (0.15.6)

[warning] Avoid specifying long messages outside the exception class

(TRY003)